spec/requests/pseuds_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe "Pseuds", type: :request do
+  subject { page }
+
+  describe "show" do
+    let(:user) { FactoryBot.create(:user, login: "Agrajag") }
+
+    before do
+      FactoryBot.create(:pseud, user: user, name: "Zaphod")
```

Comment:
Can you [leave out `FactoryBot.`](https://github.com/otwcode/otwarchive/wiki/Automated-Testing#Our_RSpec_Standards)?